kernel/kernbench.py

```diff
@@ -38,7 +39,11 @@ class Kernbench(Test):
     and compile it as per the configuration file. Performance figures will be
     shown in the log file.
     """
-
+    subprocess.call("cp /boot/config* /tmp", shell=True)
```

Comment:
using avocado library will be preferred  as more granular way as example it provide error handling internally 

example can be 
`process.system("cp /boot/config* /tmp", shell=True , sudo=true)`